frontend/src/Keyword/Keyword.module.css

```diff
@@ -1,30 +1,29 @@
-.DeleteKeyword {
+.keyword {
```

Comment:
there was a lot of repetition going on with your styles, so i've removed a lot of them and taken advantage of css's cascading nature.